graph-builder/src/graph.rs

```diff
@@ -17,13 +17,34 @@ use cincinnati::{plugins, AbstractRelease, Graph, Release, CONTENT_TYPE};
 use commons::GraphError;
 use config;
 use failure::Error;
+use prometheus::{Counter, IntGauge};
 use registry::{self, Registry};
 use serde_json;
 use std::collections::{HashMap, HashSet};
 use std::sync::{Arc, RwLock};
 use std::thread;
 
+lazy_static! {
+    static ref GRAPH_DAG_NODES: IntGauge = register_int_gauge!(
+        "cincinnati_gb_graph_dag_nodes",
+        "Number of nodes (releases) in the graph."
```

Comment:
Let's make sure this unmisunderstandably mentions "after processing".
Maybe we can rename it to `cincinnati_gb_graph_releases_processed` and explain it shortly in the description.

Reply:
I agree with that. I was trying to find a concise name to describe the "final processed DAG that is then cached as JSON-string" thing. Perhaps "processed DAG"? This would then become `graph_processed_dag_nodes` or `graph_processed_dag_releases`.

Reply:
With the same motivation as explained below, trying to find high-level names: `graph_releases_final` and for the below `graph_releases_raw`?